solution/ui/regulations/eregs-component-lib/src/components/tooltips/TooltipContainer.vue

```diff
@@ -82,41 +82,6 @@ const leftWarning = (el) => el.getBoundingClientRect().left < 130;
 export default {
     name: "TooltipContainer",
 
-    components: {},
-
-    // https://www.vuesnippets.com/posts/click-away/
-    // https://dev.to/jamus/clicking-outside-the-box-making-your-vue-app-aware-of-events-outside-its-world-53nh
-    directives: {
-        clickaway: {
```

Comment:
this custom directive is being moved to its own file so it can be injected at the App level for use in both `eregs-vite` components and `eregs-component-lib` components